File: HW6/hw6.java
public class hw6 {

	public static int adder(int ...args)
	{
		int sum = 0;
		
		for (int i : args){
			sum += i;
		}
		
		return sum;
	}
	public static void main(String [] args)
	{
		System.out.println("Sum 1 is " + adder(1,2,3,4,5,6));		
		System.out.println("Sum 2 is " + adder(1,5));		
		System.out.println("Sum 3 is " + adder());
	}
}

File: HW6/hw6.cs
using System;

namespace hw6 {
	class hw6 {
		public static int Adder(params int[] args)
		{
			int sum = 0;
			
			for (int i = 0; i < args.Length; i++){
				sum += args[i];
			}
			
			return sum;
		}
		
		public static int Main(String [] args)
		{
			Console.WriteLine("Sum 1 is {0}", Adder(1,2,3,4,5,6));
			Console.WriteLine("Sum 2 is {0}", Adder(1,5));
			Console.WriteLine("Sum 3 is {0}", Adder());
			
			return 0;
		}
	}
}
